events/conftest.py

```diff
@@ -0,0 +1,51 @@
+import pytest
+from django.utils import timezone
+from django.utils.datetime_safe import datetime
+from events.class_models.event_models import Colors
+from reminders.class_models.reminder_models import ReminderType
+
+TITLE = 'new_form'
+LOCATION = 'new_form_location'
+DESCRIPTION = 'new_form_description'
+DATE_TIME_END = "2012-12-12 12:12"
+DATE_TIME_START = "2012-11-11 11:11"
+COLOR = Colors.BLACK
+
+
+# define constants
+def pytest_configure():
+
+    pytest.current_date = datetime.now()
+
+    pytest.event_date_time_end = datetime(
+        pytest.current_date.year, pytest.current_date.month, pytest.current_date.day + 1, 14, 0, 0, tzinfo=timezone.utc
+    )
+    pytest.event_date_time_start = datetime(
+        pytest.current_date.year, pytest.current_date.month, pytest.current_date.day + 1, 13, 0, 0, tzinfo=timezone.utc
+    )
+
+    pytest.CREATE_EVENT_URL = "/event/create/"
+    pytest.CREATE_EVENT_TEMPLATE = 'events/create_event.html'
+    pytest.LOGIN_PAGE_TEMPLATE = 'login/register_login.html'
+
+    pytest.valid_event_date_time_start = DATE_TIME_START
+    pytest.valid_event_date_time_end = DATE_TIME_END
+    pytest.valid_event_color = COLOR
+    pytest.valid_event_title = TITLE
```

Comment:
Hi @RavidYael 
can you copy lines 7 - 12 into lines 31 - 34
lines 31 - 34 are designed to replace the constant like lines 7 - 12.
And please use lower case letters in this variables 😄

Reply:
about 'can you copy lines 7 - 12 into lines 31 - 34':
I believe the way it is written right now improves the maintainability of the code, is there any technical reason you suggest that?
about the lower case - done.

Reply:
done